test_graphs/individualization_refinement/bigtrees1.grl

```diff
@@ -0,0 +1,187 @@
+# Number of vertices:
```

Comment:
Ik zou deze hele commit dan ook weghalen omdat deze files er al in zitten.

Reply:
Ja mee eens, even uitzoeken hoe